src/blocks/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,131 @@
+import { ReactNode, forwardRef, useState } from 'react';
+import styled, { FlattenSimpleInterpolation } from 'styled-components';
+
+import { getBlocksColor } from 'blocks/Blocks.utils';
+import { useBlocksTheme } from 'blocks/Blocks.hooks';
+import { ModeProp } from 'blocks/Blocks.types';
+import { DropdownProps } from './Dropdown.types';
+
+const StyledDropdown = styled.div.withConfig({
+  shouldForwardProp: (prop, defaultValidatorFn) => !['mode'].includes(prop) && defaultValidatorFn(prop),
+})<DropdownProps & ModeProp>`
+  position: relative; /* Ensure the dropdown has relative positioning */
+  min-width: 200px;
+  .dropdown-item {
+    background-color: ${({ mode }) => getBlocksColor(mode, { light: 'white', dark: 'gray-900' })};
+    border: 1px solid ${({ mode }) => getBlocksColor(mode, { light: 'gray-200', dark: 'gray-800' })};
+    padding: 10px 15px;
```

Comment:
```suggestion
    padding: 10px 15px;
```
use `var(--s0)` format for gap, padding and margins

Reply:
@corlard3y only the css variables mentioned App.tsx are to be allowed.